### PR TITLE
Re-locate the Sentry error handler to take effect.

### PR DESCRIPTION
**Pull request by rtyler**
Apparently installing this errorHandler() middleware too early results in
feathers errors not being properly caught and reported.

I've verified this by running acceptance tests and seeing what gets reported in
Sentry (lots! :smile:)

Fixes JENKINS-53749

### CODE REVIEW

File: services/src/app.js [hunk 1]
```diff
@@ -49,10 +50,8 @@ logger.level = process.env.LOG_LEVEL || 'warn';
  */
 if (process.env.SENTRY_DSN) {
   logger.info('Configuring Sentry for backend errors..');
-  const Raven = require('raven');
   Raven.config(process.env.SENTRY_DSN).install();
   app.use(Raven.requestHandler());
-  app.use(Raven.errorHandler());
   logger.info('..Sentry middleware installed');
 }
```
Comment (mandie722) [inline services/src/app.js]: Is there a reason the rest of the logic wouldn't also move to the end of app startup?  It is better for code readability if all the configuration for Sentry is all together.

Comment (rtyler) [reply services/src/app.js]: Yeah, the `requestHandler` has  to be the first middleware :man_facepalming: 

It's derpy

Comment (mandie722) [reply services/src/app.js]: Super derpy

**Comment by rtyler** [comment]
@mandie722 I'm going to add some comments explaining why this is broken up a bit to avoid confusion in the future, then I'll merge

**Pull request merged**